llpc/CMakeLists.txt

```diff
@@ -183,6 +184,8 @@ target_include_directories(llpc
         translator/lib/SPIRV/libSPIRV
         util
         ../util
+        ../imported/llvm-dialects/include
```

Comment:
You can avoid that by having it as a PUBLIC entry in llvm-dialects's target_include_directories, right?